ml-proto/test/labels.wast

```diff
@@ -99,6 +99,13 @@
     )
   )
 
+  (func $redefinition (result i32)
+    (block $l1
+      (i32.add (block $l1
+		 (i32.const 2))
```

Comment:
I was referring to the inconsistent indentation and the extra line breaks in particular. That said, it's good practice for changes to adapt to surrounding styles. ;)